static/sass/_snapcraft_custom-typography.scss

```diff
@@ -48,4 +48,8 @@
   .is-ticked {
     background-position: 0 .3em;
   }
+
+  br {
```

Comment:
Would be good to include some code comment, possibly with a link to https://github.com/vanilla-framework/vanilla-framework/issues/1627 to remember where it comes from.